test/cli/test_cli_functions.sh

```diff
@@ -668,3 +668,37 @@ function check_backingstore {
     echo_time "💬  Deleting bucket testbucket"
     test_noobaa bucket delete "testbucket"
 }
+
+function check_dbdump {
+    # Generate db dump
+    echo_time "💬  Generaiting db dump"
+    test_noobaa db-dump --dir /tmp
+
+    # Check whether dump was created
+    local dump=`ls -l /tmp | grep noobaa_db_dump | awk '{ print $9 }'`
+    if [ ! -f "$dump" ]
```

Comment:
Now it is not correct,  you are doing `ls -l` you will not get only the file name and you will also not get the path.

Reply:
Same as above.